src/progress_bar.rs

```diff
@@ -517,6 +515,26 @@ impl ProgressBar {
         self.state().draw_target.remote().map(|(_, idx)| idx)
     }
 
+    /// Current message
```

Comment:
I don't see a good reason to add all this public API?

Reply:
I added these because I need the ability to re-style the prefix and message at runtime. I can make them a separate PR if you'd like.

Reply:
Well, I think the `unstyled()` getters are very niche and you should handle that in your project. For `message()` and `prefix()`, can we just return a `&str`?

Reply:
Sure, I'll remove the `unstyled()` getters. We can't return a `&str` because it's behind a lock.

Reply:
Ah yes... too bad std doesn't have guard mapping like Tokio.